com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/DeleteDiscontinuedConfigurationEntriesForAppStep.java

```diff
@@ -92,24 +106,8 @@ private DeployedMtaApplication getDeployedMtaApplication(CloudApplication existi
         return envMtaMetadataParser.parseDeployedMtaApplication(existingApp);
     }
 
-    @Override
-    protected String getStepErrorMessage(ProcessContext context) {
-        return MessageFormat.format(Messages.ERROR_DELETING_DISCONTINUED_CONFIGURATION_ENTRIES_FOR_APP,
-                                    context.getVariable(Variables.EXISTING_APP)
-                                           .getName());
-    }
-
-    private List<ConfigurationEntry> getEntriesToDelete(String mtaId, String mtaVersion, CloudTarget target,
-                                                        List<String> providedDependencyNames, List<ConfigurationEntry> publishedEntries) {
-        List<ConfigurationEntry> entriesForCurrentMta = getEntries(mtaId, mtaVersion, target);
-        List<ConfigurationEntry> entriesForCurrentModule = getConfigurationEntriesWithProviderIds(entriesForCurrentMta,
-                                                                                                  getProviderIds(mtaId,
-                                                                                                                 providedDependencyNames));
-        return getEntriesNotUpdatedByThisProcess(entriesForCurrentModule, publishedEntries);
-    }
-
-    private List<ConfigurationEntry> getEntriesNotUpdatedByThisProcess(List<ConfigurationEntry> entriesForCurrentModule,
-                                                                       List<ConfigurationEntry> publishedEntries) {
+    private List<ConfigurationEntry> getConfigEntriesNotUpdatedByThisProcess(List<ConfigurationEntry> entriesForCurrentModule,
```

Comment:
Inconsistent naming. It should be "configuration entries" and not "config entries".